common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -276,21 +269,22 @@ status_function() {
     pgrep -l ieee1905_transport
     pgrep -l local_bus
 
-    # check for operational status
-    LOGS_PATH=@BEEROCKS_LOG_FILES_PATH@
+    bridge_mac="$(ifconfig @BEEROCKS_BRIDGE_IFACE@ | grep -o -E '([[:xdigit:]]{1,2}:){5}[[:xdigit:]]{1,2}')"
```

Comment:
In dockers we use a slightly different command for extracting the bridge mac address:
`bridge_mac="$(ip link show dev br-lan | awk '/^ *link\/ether / {print $2}')"`

To me it looks a bit less complex.